project/build.scala

```diff
@@ -125,12 +136,12 @@ object BananaRdfBuild extends Build {
   lazy val rdfTestSuite_jvm = rdfTestSuiteM
     .project(Jvm, rdfTestSuite_common_jvm)
     .settings(Seq(resourceDirectory in Test := baseDirectory.value / "src/main/resources"):_*)
-    .dependsOn(rdf_jvm)
+    .dependsOn(rdf_jvm, ntriples_jvm)
```

Comment:
Done. Anything else required?

P.S. I just want to emphasize that while build is broken in IDEA/Eclipse we keep losing potential b-rdf users as they fail to import b-rdf to their IDEs...